eng/pipelines/common/platform-matrix.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       archType: arm
       platform: Linux_arm
       container:
-        image: ubuntu-16.04-cross-14.04-23cacb0-20200121150126
+        image: ubuntu-18.04-cross-arm-16.04-09ec757-20200324125113
```

Comment:
Why did we need to upgrade the image here?

Reply:
The script needs python3.6+

Reply:
Its also generally not needed for us to target 14.04 anymore as 14.04 is eol